crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -101,5 +101,8 @@ fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
 #ifdef DEBAND_DITHER
         output_color = dither(output_color, in.frag_coord.xy);
 #endif
+
+    output_color = premultiply_alpha(material.flags, output_color);
```

Comment:
I feel like this should be behind a shader def. It's probably not that expensive, but still, when the mesh is opaque it doesn't need to be in the shader.

Reply:
Moved it to behind a shader def